### PR TITLE
Generate baked data for Units Display Names

### PR DESCRIPTION
**Pull request by younies**

### CODE REVIEW

File: components/experimental/src/dimension/provider/units.rs [hunk 1]
```diff
@@ -12,6 +12,16 @@
...
 
+#[cfg(feature = "compiled_data")]
+/// Baked data
+///
+/// <div class="stab unstable">
+/// 🚧 This code is considered unstable; it may change at any time, in breaking or non-breaking ways,
+/// including in SemVer minor releases. In particular, the `DataProvider` implementations are only
+/// guaranteed to match with this version's `*_unstable` providers. Use with caution.
+/// </div>
+pub use crate::provider::Baked;
```
Comment (robertbastian) [inline components/experimental/src/dimension/provider/units.rs:23]: you shouldn't need this, you can always refer to the provider as `crate::provider::Baked`

Comment (younies) [reply components/experimental/src/dimension/provider/units.rs]: done

File: components/experimental/src/lib.rs [hunk 2]
```diff
@@ -91,6 +91,7 @@ pub mod provider {
         super::compactdecimal::provider::ShortCompactDecimalFormatDataV1Marker::INFO,
         super::dimension::provider::currency::CurrencyEssentialsV1Marker::INFO,
         super::dimension::provider::percent::PercentEssentialsV1Marker::INFO,
+        super::dimension::provider::units::UnitsDisplayNameV1Marker::INFO,
```
Comment (robertbastian) [inline components/experimental/src/lib.rs:94]: issue: you need to wire up the impl with `impl_units_display_name_v1_marker!(Baked)`

Comment (younies) [reply components/experimental/src/lib.rs:94]: Sure, I added it in #5000

**Pull request merged**